src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,10 +35,13 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final Shooter shooterSubsystem = new Shooter();
+  private final Intake intakeSubsystem = new Intake();
+
+  
   private final AmpCommand ampCommand = new AmpCommand(shooterSubsystem);
   private final SpeakerCommand speakerCommand = new SpeakerCommand(shooterSubsystem);
-  private final ShootCommand shootCommand = new ShootCommand(shooterSubsystem);
-  private final Intake intakeSubsystem = new Intake();
+  private final ShootCommand shootCommand = new ShootCommand(shooterSubsystem, intakeSubsystem);
+  private final Eject eject = new Eject(intakeSubsystem, -0.2);
```

Comment:
All of these have `<name>Command` and this is just `<name>`. Should be `ejectCommand`.